src/components/FileDrop.vue

```diff
@@ -141,14 +143,19 @@ export default {
 .errormessage {
 	margin-top: 1.25rem;
 	color: var(--red);
```

Comment:
Instead of `--red` the _meaning_ is preferred (i.e. `--error`). Unrelated in this PR.

Reply:
I would normally not describe a variable like that, indeed, but since these are _the_ Wakamai Fondue Brand Colors™ I found it acceptable to use 'em like this!